tests/blueprints/.eslintrc.cjs

```diff
@@ -0,0 +1,43 @@
+const imports = require('@warp-drive/internal-config/eslint/imports.cjs');
+const parser = require('@warp-drive/internal-config/eslint/parser.cjs');
+const isolation = require('@warp-drive/internal-config/eslint/isolation.cjs');
+const ignore = require('@warp-drive/internal-config/eslint/ignore.cjs');
+const node = require('@warp-drive/internal-config/eslint/node.cjs');
+const base = require('@warp-drive/internal-config/eslint/base.cjs');
+
+module.exports = {
+  ...parser.defaults(),
+
+  plugins: [...base.plugins(), ...imports.plugins()],
+  extends: [...base.extend()],
+  rules: Object.assign(
+    base.rules(),
+    imports.rules(),
+    // { allowedImports: ['@ember/debug', 'ember-inflector', '@ember/string'] }
+    isolation.rules(),
+    {}
+  ),
+
+  ignorePatterns: ignore.ignoreRules(),
+
+  overrides: [
+    node.defaults({
+      useModules: true,
+      files: ['tests/**/*.{js,ts}'],
+      env: {
+        mocha: true,
+      },
+      rules: {
+        'n/no-unpublished-require': 'off',
+      },
+    }),
+    node.defaults({
+      useModules: true,
+      files: ['fixtures/**/*.{js,ts}'],
+      rules: {
+        'import/order': 'off',
+        'simple-import-sort/imports': 'off',
```

Comment:
Should we just autofix these instead of turning them off?

Reply:
iirc it doesnt work right for node files is why I turned them off, but that was a long time ago.